classes/class3/homework/views/oldcat.handlebars

```diff
@@ -0,0 +1,6 @@
+<h1>Say Bye bye to:</h1>
```

Comment:
You could have creatively combined `oldcat.handlebars` and `newcat.handlebars` by making the title a variable.